### ISSUE TITLE
Instructions on running game with modified .txts?

### ISSUE THREAD

**Issue by epleaner** [Lukaszpg/PD2-Single-Player-Plus-mod#12]
Hi, thanks for this mod!

If I wanted to further make some changes to the .txt files and not use the compiled "pd2data.mpq" file, how would I go about doing that?

**Comment by Lukaszpg** [comment]
Hey, thanks for checking out the mod. You have to unpack the pd2data.MPQ file with MPQExtractor, then put the extracted **data** folder into **ProjectD2** directory and add -direct -txt to your PlugY shortcut.

**Comment by epleaner** [comment]
Thanks for the response. I've followed your instructions, and have the /data folder extracted from pd2data.MPQ placed in the ProjectD2 directory, and I've added -direct -txt to my PlugY shortcut (Target looks like` "C:\Program Files (x86)\Diablo II - SP\ProjectD2\PlugY.exe" -3dfx -direct -txt`). However, anytime I try to enter into a game I get an `Access Violation (c0000005)`. I haven't made any .txt modifications yet.

Mod runs smoothly without -direct -txt.
